app/stacks/uk-west/applications-service/terragrunt.hcl

```
@@ -55,6 +56,7 @@ inputs = {
   back_office_service_bus_register_nsip_subscription_topic_id   = dependency.back_office_ukw.outputs.servicebus_topic_register_nsip_subscription_id
   back_office_service_bus_nsip_project_update_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_project_update_topic_id
   back_office_service_bus_nsip_representation_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_representation_id
+  back_office_service_bus_nsip_exam_time_table_topic_id         = dependency.back_office_ukw.outputs.service_bus_nsip_exam_time_table_id
```

naming on the left side here needs to match what is defined in `app/stacks/{region}/applications-service/variables.tf`, as below:

```suggestion
  back_office_service_bus_nsip_exam_timetable_topic_id         = dependency.back_office_ukw.outputs.service_bus_nsip_exam_time_table_id
```

---

I think github has marked this as outdated/resolved, but still showing as the old value to me

---

Ah I tried to use the "accept suggestion" button and thought it worked. Good catch! Done manually now.